SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -4,6 +4,7 @@
 
 #include "Interop/CrossServerRPCHandler.h"
 #include "Engine/EngineBaseTypes.h"
+#include "EngineClasses/SpatialShadowActor.h"
```

Comment:
i forget the rules, could this be forward declared?

Reply:
Yes